extensions/lifecycle/conductor/LifecycleConductor.js

```diff
@@ -0,0 +1,266 @@
+'use strict'; // eslint-disable-line
+
+const async = require('async');
+const schedule = require('node-schedule');
+const zookeeper = require('node-zookeeper-client');
+
+const Logger = require('werelogs').Logger;
+
+const BackbeatProducer = require('../../../lib/BackbeatProducer');
+const zookeeperHelper = require('../../../lib/clients/zookeeper');
+
+const DEFAULT_CRON_RULE = '* * * * *';
```

Comment:
Maybe these aren't needed as the joi configs either set a default value already or the field is required

Reply:
Right, but they are meant to be used in cases where the params may not have been parsed by joi (e.g. for testing), as the class is supposed to be usable standalone from the config parser, so these params are meant to be the class defaults.